lib/ansible/modules/cloud/azure/azure_rm_containerinstance.py

```diff
@@ -93,7 +93,7 @@
         description:
             - Force update of existing container instance. Any update will result in deletion and recreation of existing containers.
         type: bool
-        default: False
+        default: 'no'
```

Comment:
Why the false is incorrect?

Reply:
not sure, just merged from newer version.... i think it can be removed